simulatingrisk/hawkdovemulti/model.py

```diff
@@ -194,6 +212,52 @@ def __init__(
         # store whether to compare cumulative payoff or since last adjustment round
         self.adjust_payoff = adjust_payoff
 
+    def _risk_level_in_bounds(self, value):
+        # check if a generated risk level is within bounds
+        return self.min_risk_level <= value <= self.max_risk_level
+
+    def get_risk_attitude_generator(self):
+        """return a generator that will return risk attitudes for individual
+        agents based on the configured distribution."""
+        if self.risk_distribution == "uniform":
+            # uniform/random: generate random integer within risk level range
+            while True:
+                yield self.random.randint(self.min_risk_level, self.max_risk_level)
+        if self.risk_distribution == "normal":
+            # return values from a normal distribution centered around 4
+            while True:
+                yield int(self.random.gauss(4, 1.5))
```

Comment:
Looks good to me. Cool to have two kinds of "normal" here compared (normal `random` behavior, normal/Gaussian distribution).

Reply:
This should be centered around 4.5.